src/Frontend/Modules/Search/Layout/Widgets/Form.html.twig

```diff
@@ -6,9 +6,13 @@
         {{ 'lbl.SearchTerm'|trans|ucfirst }}<abbr title="{{ 'lbl.RequiredField'|trans|ucfirst }}">*</abbr>
       </label>
       <div class="form-group">
-        {% form_field q_widget %}
+        <div class="input-group">
+          {% form_field q_widget %}
+          <div class="input-group-btn">
+            <input id="submit" class="btn btn-default" type="submit" name="submit" value="{{ 'lbl.Search'|trans|ucfirst }}" />
```

Comment:
Is this id necessary? Also remove the `|ucfirst` if you don't want capitals in input-group-btn's